service/service_test.go

```diff
@@ -106,7 +105,8 @@ func TestRun(t *testing.T) {
 			}
 			svcErrors := make(chan error, 1)
 			svcList := service.NewServiceList(initMock)
-			_, err := service.Run(ctx, svcList, testBuildTime, testGitCommit, testVersion, svcErrors)
+			cfg, _ := config.Get()
```

Comment:
Eventhough it is a test we should handle error and fail test with message of why it failed

Same for calling `service.Run`

Check other tests too; thanks